cafe.rb

```diff
@@ -19,7 +19,7 @@ def take_order(menus)
     puts "(#{i})#{menu[:name]}: #{menu[:price]}円"
   end
   print '>'
-  order_number = gets.to_i
+  order_number = gets.to_i-1
```

Comment:
```suggestion
  order_number = gets.to_i - 1
```

のように演算子の前後にはスペースを入れましょう。